client/stylesheets/_responsive.scss

```diff
@@ -93,6 +131,37 @@
     .homeCardImg {
         height: 190px !important;
     }
+
+    .searchResultsCardImg {
+        height: 155px !important;
+    }
+
+    .searchResultsCardPrice {
+        margin: 6px 0 8px 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `6px 0 8px` instead of `6px 0 8px 0`